package.json

```diff
@@ -11,38 +11,35 @@
     "ng": "ng",
     "start": "NG_PERSISTENT_BUILD_CACHE=1 ng serve",
     "build": "NG_PERSISTENT_BUILD_CACHE=1 ng build --configuration production",
-    "build:common": "ng build common",
-    "build:components": "ng build components",
-    "build:dashboards": "ng build dashboards",
     "build:ci": "node --max_old_space_size=3584 node_modules/@angular/cli/bin/ng build --configuration production --no-progress",
-    "test": "ng test hypertrace-ui --cache --maxWorkers=2",
+    "test": "ng test hypertrace-ui --cache --max-workers=2",
     "lint": "ng lint hypertrace-ui",
     "lint:fix": "ng lint --fix hypertrace-ui",
     "prettier:check": "prettier --check '**'",
-    "test:ci": "ng test hypertrace-ui --maxWorkers=2 --ci --coverage",
+    "test:ci": "ng test hypertrace-ui --ci --coverage",
```

Comment:
intentionally removing max workers here?

Reply:
It is not needed for ci environment because we only have 1 or 2 CPU machines there. I can bring it back if you prefer.

Reply:
reverted

Reply:
It was actually originally introduced for CI - it's possible this isn't the case any more, but historically it was getting the CPU count from the underlying hardware, while the actual container running the job had less CPUs available, causing a big slowdown. In GHA, since we're running on VMs rather than in docker it may not be relevant, so you could try opening it up and see how times compare but wouldn't do it in this PR.

Reply:
Reverted it and it also fixed the weird test failure. :)